SKA-ScienceDataProcessor/integration-prototype
# TSK-2727 Initial Tango Log aggregator

## Description:
Provide an initial "Log aggregator" for the SIP Tango Device Servers - using the Master DS as an example

TSK-2727 SIP: Add aggregated Tango log messages from the SIP Tango Master device

## Testing instructions:

From the top level sip repository:

1. Install/update the Tango SIP stack modules:
```bash
docker stack deploy - c sip/tango_control/tango_logger/docker_compose.yml sip
```

2. Bring up an iTango prompt to the current Tango stack
```bash
./deploy/demos_23_11_18/scripts/get_itango_prompt
```

3. Type the following (to the iTango prompt from (2)) to get the status of the Master Device Server
```python
d=DeviceProxy('sip_sdp/elt/master')
d.status()
quit
```

4. examine Docker logs for Logger
```bash
docker service logs $(docker stack services -q --filter name=sip_tc_tango_logger sip)
```

5. Expect a Tango logging message from Tango_master received by Tango logger!!!!

## Types of changes


## Checklist:
